scripts/startup/tar/linux/opensearch-tar-install.sh

```diff
@@ -10,7 +10,10 @@ cd $OPENSEARCH_HOME
 KNN_LIB_DIR=$OPENSEARCH_HOME/plugins/opensearch-knn/lib
 ##Security Plugin
 if [ -d "$OPENSEARCH_HOME/plugins/opensearch-security" ]; then
-        bash $OPENSEARCH_HOME/plugins/opensearch-security/tools/install_demo_configuration.sh -y -i -s
+        echo "OpenSearch 2.11.0 onwards, the security plugin introduces a change that requires an initial password for 'admin' user."
+        echo "Please define an environment variable 'initialAdminPassword' with a password string."
+        echo “Or create a file 'initialAdminPassword.txt' with a single line that contains the password string and place it under $OPENSEARCH_PATH_CONF folder.”
+        bash $OPENSEARCH_HOME/plugins/opensearch-security/tools/install_demo_configuration.sh -y -i -s || exit 1
```

Comment:
What happens if `plugins.security.authcz.admin.password` setting is set? @RyanL1997 
Does it set the `initialAdminPassword` env variable so the check in `tools/install_demo_configuration.sh` doesn't fail?

Reply:
Hi @rishabh6788, thanks for the feedback. Yes, so if that setting is set, the security plugin will use that password, and yes it will run the install demo config script.

Reply:
So would that plugin setting set the `initialAdminPassword` env variable?

Because as per [this](https://github.com/opensearch-project/security/blob/main/tools/install_demo_configuration.sh#L395-L401), if the code doesn't find the `initialAdminPassword` env set or `ADMIN_PASSWORD_FILE="$OPENSEARCH_CONF_DIR/initialAdminPassword.txt"` file then it is going to error out.

Reply:
Or is the plugin setting adding password to `$OPENSEARCH_CONF_DIR/initialAdminPassword.txt` file at runtime?

Reply:
> Because as per [this](https://github.com/opensearch-project/security/blob/main/tools/install_demo_configuration.sh#L395-L401), if the code doesn't find the initialAdminPassword env set or ADMIN_PASSWORD_FILE="$OPENSEARCH_CONF_DIR/initialAdminPassword.txt" file then it is going to error out.

I think this is what we are expecting, since we would like to force user to setup the initial admin password instead of using the `admin:admin` So to make the long story short, no more admin:admin, and we want to fail both demo config setup and initiation of the cluster if there is an absent of this initial admin password provided by user.

Reply:
FYI, I am not able to find `plugins.security.authcz.admin.password` setting in Security plugin code. Was this implemented? @RyanL1997

Reply:
@rishabh6788 Sorry for the confusions. No this is not implemented. I will re-edit the description of that issue and original PR, and this was some stale design. I think the current change we are having right now should be only in the install of demo configuration script.

Reply:
There is no such setting in the opensearch.yml. The only way to set admin password on boot-up with the new change is to set it via demo installation script.